chunky3d/helpers.py

```diff
@@ -1,47 +1,118 @@
 import numpy as np
 
 
+def adjust_key(key, shape):
+    """Prepare a slicing tuple (key) for slicing an array of specific shape."""
+
+    if type(key) is not tuple:
+        key = (key,)
```

Comment:
```suggestion
        key = tuple(key)
```
In case somebody tried to `__setitem__` using a list instead of tuple.

Reply:
No, this is correct. it could also be `tuple([key])` - 1-element tuple, but I went for the literal.

Setitem using a list could be "fancy indexing": https://jakevdp.github.io/PythonDataScienceHandbook/02.07-fancy-indexing.html - we may think  about it in the future, but it requires some extra work.

The reason I do it is because if someone passes `x[2, 3]` then key is `(2, 3)`, but for `x[4]` the key is just `4`, without the tuple. And then I couln't:
- iterate over key dimiensions
- check `if key.count(...) > 0`